src/common/components/form/fields/PlaceSelectorField.tsx

```diff
@@ -0,0 +1,69 @@
+import { FieldProps } from 'formik';
+import { DropdownProps } from 'hds-react';
+import React from 'react';
+
+import PlaceSelector from '../../../../domain/place/placeSelector/PlaceSelector';
+
+export type Option = {
+  label: string;
+  value: string;
+};
+
+interface Props extends DropdownProps, FieldProps {
+  options: Option[];
+  title: string;
+  inputPlaceholder: string;
+  setFieldValue?: (
```

Comment:
Can you import setFieldValue type from formik? If not you could create a re-usable type. I think that this is also defined on DropdownField component